manifests/init.pp

```diff
@@ -3,6 +3,7 @@
   String $ensure                         = present,
   Boolean $hieramerge                    = false,
   Boolean $manage_cron_daily             = true,
+  Boolean $manage_cron_hourly            = true,
```

Comment:
This would be a breaking change, because a before unmanaged file would now be managed and might get purged. In my opintion this should default to ```false```

Reply:
Happy to change the default to false

Reply:
That being said in the current code logic if a rule is put in place for hourly then it includes ::logrotate::hourly (note the ::) which automatically added the /etc/cron.hourly entry.

So it automagically managed cron hourly but again if you changed your rule away from hourly it never cleaned up the /etc/cron.hourly entry.

Unless I misundertand the -> operator used.

Reply:
If the default is left to true yes it creates a /etc/cron.hourly entry when there may not be any hourly rules, but logrotate will run without erroring it just won't process anything.